docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,131 @@
 
 # Quickstart Guide
 
+## Dependencies
+
+Please ensure you have the following dependencies installed:
+
+* [gen3-client](../requirements/gen3-client.md)
+* [gen3-tracker](../requirements/gen3-tracker.md)
+
+## Usage
+
+`g3t  [OPTIONS] COMMAND [ARGS]...`
+
+The following options and environmental variables are synonymous, you may set them as environmental variables or pass them as parameters to the command line.
+
+| option       | environment     | comment             | example       |
+|--------------|-----------------| ------------------- |---------------|
+| --project_id | G3T_PROJECT_ID  | authorization       | aced-test     |
+| --profile    | G3T_PROFILE     | gen3-client profile | production    |
+| --format     | G3T_FORMAT      | Output format.      | default: yaml |
+
+Alternatively, you can set the environmental variables using the `EXPORT` function e.g.:
+
+```sh
+export G3T_PROJECT_ID=aced-myproject
+```
+
+
 ## 1. Upload Data (To an Existing Project on ACED IDP)
-This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarily to `git` commands
+This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarly to `git` commands
```

Comment:
Add something about how users need to be granted permissions to a project on ACED? (hence explaining why it has to be an existing project?)

(might have missed this being mentioned somewhere else)

Reply:
mentioned somewhere else

Reply:
https://deploy-preview-23--luminous-boba-a6e270.netlify.app/workflows/add-users/